applications/debug/example_custom_font/example_custom_font.c

```diff
@@ -0,0 +1,104 @@
+#include <furi.h>
+#include <furi_hal.h>
+
+#include <gui/gui.h>
+#include <input/input.h>
+
+
+//This include enables font usage
+
+#include <u8g2_glue.h>
```

Comment:
This is a private header that is not exposed in SDK. This app is not buildable with ufbt.

Reply:
Fixed.